app/models/solr_document.rb

```diff
@@ -2,27 +2,30 @@
 
 class SolrDocument
   include Blacklight::Solr::Document
-  # The following shows how to setup this blacklight document to display marc documents
```

Comment:
oh ha! I just noticed this in my latest PR on BL wondering if we could delete. Nice :)

Reply:
oh I see you just moved it down. We do need it then? Is this what produces the librarian view?

Reply:
yeah I just moved it down

Reply:
and yes I think it is for librarian view